addons/test_website/tests/test_fuzzy.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo.addons.website.controllers.main import Website
+from odoo.addons.website.tools import MockRequest
+import odoo.tests
+from odoo.tests.common import TransactionCase
+
+_logger = logging.getLogger(__name__)
+
+@odoo.tests.tagged('-at_install', 'post_install')
+class TestAutoComplete(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.website = cls.env['website'].browse(1)
+        cls.WebsiteController = Website()
+
+    def _autocomplete(self, term, expected_count, expected_term):
```

Comment:
The expected term is just the expected fuzzy term, right?
```suggestion
    def _autocomplete(self, term, expected_count, expected_fuzzy_term):
```